packages/bento-design-system/src/DefaultMessagesContext.tsx

```diff
@@ -27,6 +27,10 @@ export type DefaultMessages = {
     Loader: {
       loadingMessage: LocalizedString;
     };
+    DateField: {
+      previousMonthLabel: LocalizedString;
+      nextMonthLabel: LocalizedString;
+    };
```

Comment:
da aggiungere ai vari `defaultMessages.ts` in giro (incluso quello nella config). By the way potremmo importare quello nella config e non duplicarlo...